opt/scopeserver/dataserver/utils/search.py

```diff
@@ -131,8 +131,10 @@ def aggregate_matches(matches: List[MatchResult]) -> Dict[Tuple[str, str], List[
 
 
 def create_feature_description(
-    aggregated_matches: Dict[Tuple[str, str], List[str]], features: Dict[Tuple[str, str], str],
-) -> Dict[Tuple[str, str], str]:
+    aggregated_matches: Dict[Tuple[str, str], List[str]],
+    features: Dict[Tuple[str, str], str],
+    feature_types: Dict[Tuple[str, str], str],
+) -> Tuple[Dict[Tuple[str, str], str], Dict[Tuple[str, str], str], Dict[Tuple[str, str], str]]:
```

Comment:
Difficult to understand the structure of the `Dict`. Would be nice in the future if these `Tuple` were informative with custom types.